### PR TITLE
More changes to allow Geyser to cooperate better with bad internet

### PR DESCRIPTION
**Pull request by Camotoy**

### CODE REVIEW

File: connector/src/main/java/org/geysermc/connector/utils/LocaleUtils.java [hunk 1]
```diff
@@ -168,9 +174,13 @@ private static void downloadLocale(String locale) {
...
-        // Get the hash and download the locale
-        String hash = ASSET_MAP.get("minecraft/lang/" + locale + ".json").getHash();
-        WebUtils.downloadFile("https://resources.download.minecraft.net/" + hash.substring(0, 2) + "/" + hash, localeFile.toString());
+        try {
+            // Get the hash and download the locale
+            String hash = ASSET_MAP.get("minecraft/lang/" + locale + ".json").getHash();
+            WebUtils.downloadFile("https://resources.download.minecraft.net/" + hash.substring(0, 2) + "/" + hash, localeFile.toString());
+        } catch (Exception e) {
+            GeyserConnector.getInstance().getLogger().error("Unable to download locale file hash", e);
+        }
```
Comment (FBioCS1) [inline connector/src/main/java/org/geysermc/connector/utils/LocaleUtils.java:183]: It's supposed to solve the issue.

**Review by FBioCS1 — COMMENTED** [review]
Pull it.

**Review by Tim203 — APPROVED** [review]

**Comment by FBioCS1** [comment]
It still dosen't work, cause it doesn't throw any exception, it still stuck there.

**Comment by FBioCS1** [comment]
It can start up when I compeletely disabled the network. But when I connect to Internet, the locale task is keeping alive and stuck there.

**Pull request merged**